src/library/scala/collection/BitSetLike.scala

```diff
@@ -15,51 +15,57 @@ package collection
 
 import BitSetLike._
 import mutable.StringBuilder
+import scala.util.hashing.MurmurHash3
+import java.lang.{Long => jLong}
+import java.util
+
+import scala.runtime.AbstractFunction1
 
 /** A template trait for bitsets.
- *  $bitsetinfo
+ * $bitsetinfo
  *
  * This trait provides most of the operations of a `BitSet` independently of its representation.
  * It is inherited by all concrete implementations of bitsets.
  *
- *  @tparam  This the type of the bitset itself.
- *
- *  @define bitsetinfo
- *  Bitsets are sets of non-negative integers which are represented as
- *  variable-size arrays of bits packed into 64-bit words. The memory footprint of a bitset is
- *  determined by the largest number stored in it.
- *  @author  Martin Odersky
- *  @since 2.8
- *  @define coll bitset
- *  @define Coll `BitSet`
+ * @tparam  This the type of the bitset itself.
+ * @define bitsetinfo
+ * Bitsets are sets of non-negative integers which are represented as
+ * variable-size arrays of bits packed into 64-bit words. The memory footprint of a bitset is
+ * determined by the largest number stored in it.
+ * @author Martin Odersky
+ * @since 2.8
+ * @define coll bitset
+ * @define Coll `BitSet`
  */
-trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSetLike[Int, This] { self =>
+trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSetLike[Int, This] {
+  self =>
 
   def empty: This
+  protected[this] def copyOrSelf: This = fromBitMaskNoCopy(toBitMask)
```

Comment:
This should `private[scala]` to justify the MiMa exception. Otherwise a new user-defined subclass of `BitSetLike` could call this and contain a reference to the new API that isn't available in 2.12.12 and below.

Reply:
done